.github/DISCUSSION_TEMPLATE/gsoc-2025-proposal-reviews.yml

```diff
@@ -1,9 +1,9 @@
-title: "GSoC’23 Proposal Review | {{Project Name}} ({{Your Name}})"
+title: "GSoC’25 Proposal Review | {{Project Name}} ({{Your Name}})"
 body:
 - type: input
   attributes:
-    label: Google doc link to Proposal
-    description: "Please paste the google doc link to the proposal here."
+    label: Google Doc link to Proposal
```

Comment:
Sorry, one more small change: either "proposal" should be lowercase, or both "Link" and "Proposal" should be uppercase.

Reply:
Done